docker/addons/vault/README.md

```diff
@@ -45,22 +45,22 @@ When the Vault service is started, some initialization steps need to be done to
 
 ## Setup
 
-The following scripts are provided, which work on the running Vault service in Docker.
+The following scripts are provided, which work on the running Vault service from within the `docker/addons/vault/scripts` directory.
 
 ### 1. `vault_init.sh`
 
-Calls `vault operator init` to perform the initial vault initialization and generates a `docker/addons/vault/data/secrets` file which contains the Vault unseal keys and root tokens.
+Calls `vault operator init` to perform the initial vault initialization and generates a `docker/addons/vault/scripts/data/secrets` file which contains the Vault unseal keys and root tokens.
 
 Example contents for `data/secrets`:
 
 ```bash
-Unseal Key 1: Ay0YZecYJ2HVtNtXfPootXK5LtF+JZoDmBb7IbbYdLBI
-Unseal Key 2: P6hb7x2cglv0p61jdLyNE3+d44cJUOFaDt9jHFDfr8Df
-Unseal Key 3: zSBfDHzUiWoOzXKY1pnnBqKO8UD2MDLuy8DNTxNtEBFy
-Unseal Key 4: 5oJuDDuMI0I8snaw/n4VLNpvndvvKi6JlkgOxuWXqMSz
-Unseal Key 5: ZhsUkk2tXBYEcWgz4WUCHH9rocoW6qZoiARWlkE5Epi5
+Unseal Key 1: `<UNSEAL_KEY_1>`
+Unseal Key 2: `<UNSEAL_KEY_2>`
+Unseal Key 3: `<UNSEAL_KEY_3>`
+Unseal Key 4: `<UNSEAL_KEY_4>`
+Unseal Key 5: `<UNSEAL_KEY_5>`
```

Comment:
Revert this change. While placeholders are self-explanatory, this conflicts with the `example` premise (i.e. this is not an example anymore and tells nothing about how keys look like).